ci-operator/config/openshift/installer/openshift-installer-release-4.5__rc.yaml

```diff
@@ -0,0 +1,35 @@
+resources:
```

Comment:
This is in the wrong location.  If this is going to be used by release controller, it should be in:

ci-operator/config/openshift/release/openshift-release-master__release-4.5.yaml

and your `--variant` should be `release-4.5`.

If you have problems with the period in the variant, call it `release-4-5`.

Reply:
The content in `ci-operator/config/openshift/release/openshift-release-master__release-4.5.yaml` is totally different, I don't follow this.

The file here is a workaround proposed to us to make multistage work in the release controller, at least that's my understanding of it.